cel/io.go

```diff
@@ -126,6 +126,37 @@ func ValueAsAlphaProto(res ref.Val) (*exprpb.Value, error) {
 	return alpha, err
 }
 
+// RefValToExprValue converts between ref.Val and cel.expr.ExprValue.
+// The result ExprValue is the serialized proto form.
+func RefValToExprValue(res ref.Val) (*celpb.ExprValue, error) {
```

Comment:
Would you mind updating this method to return v1alpha1.ExprValue as the output? The convention is as follows:

```
RefValToExprValue // v1alpha1
ValueAsAlphaExprValueProto // v1alpha1
ValueAsExprValueProto // canonical
```

The alpha proto implementations proxy into the canoncial value and convert the type to v1alpha1before returning.